content/en/agent/faq/certificate_verify_failed-error.md

```diff
@@ -68,27 +70,36 @@ rm "C:\Program Files\Datadog\Datadog Agent\files\datadog-cert.pem"
 restart-service -Force datadogagent
 ```
 
-*Windows GUI*
+If your Agent is configured to use a proxy, see the additional step in the [dedicated section below](#additional-step-if-the-windows-agent-5-x-is-configured-to-use-a-proxy-or-the-curl-http-client).
+
+*Using the Windows GUI*
+
+Delete `datadog-cert.pem`. You can locate this file in:
+
+* Agent `>=5.12.0`:
+  * 64-bit Windows: `C:\Program Files\Datadog\Datadog Agent\agent\`
+  * 32-bit Windows: The Datadog Agent was not available for 32-bit Windows systems starting in Agent 5.12
+* Agent `<= 5.11.x`:
+  * 64-bit Windows: `C:\Program Files (x86)\Datadog\Datadog Agent\files\`
```

Comment:
Is `C:\Program Files (x86)\Datadog\Datadog Agent\files\` correct for 64-bit Windows? 
Usually, `C:\Program Files (x86)\Datadog\Datadog Agent\files\` contains 32-bit applications.